### ISSUE TITLE
Database migration needs to be created for the assessments feature structure

### ISSUE THREAD

**Issue by seidior** [OpenTree-Education/rhizone-lms#504]
### Describe the Feature / Enhancement

A new database migration needs to be created that contains the `CREATE TABLE` and `ALTER TABLE` commands that will set up the tables for the assessments feature.

### Additional Details and Resources

Link to ERD: https://drawsql.app/teams/opentree-education/diagrams/assessments

### Correct Project Selected

- [X] This issue is linked to the appropriate project

### Labels

- [X] This issue has appropriate labels.

**Comment by seidior** [comment]
EOD Update: I created the branch (sorry, team!) as an example to the other groups for what they have to do in their issues, and I committed the migration file as an example. The team has begun work on crafting the SQL syntax necessary for creating the new tables (green in ERD) and altering the existing ones (teal in ERD) to complete the ticket. There are some existing tables whose changes we discussed verbally that I will need to share with them tomorrow in order to fully complete this task, but they have their work cut out for them for now.

Next Steps:
- [ ] Use Sequel Ace / mySQL Workbench to craft the `CREATE TABLE` / `ALTER TABLE` syntax for each table, then copy each final SQL command to the migration file.
- [ ] Test the migration file to ensure the database structure after migrations is as expected.

Out of Scope:
- Inserting dummy data inside the tables
- Anything related to `api` or `webapp`
